### PR TITLE
Python activities issues OSAC 39,40,41,50

### PR DESCRIPTION
**Pull request by petcua1**
The following issues have been resolved:
https://github.com/UiPath/Community.Activities/issues/34 - OSAC-40
https://github.com/UiPath/Community.Activities/issues/35 - OSAC-41
https://github.com/UiPath/Community.Activities/issues/36(Which is related to #35) - OSAC-39
https://github.com/UiPath/Community.Activities/issues/38 - OSAC-50
Now the Auto version is inferred from running python --version in the folder where Python is installed 
In case of mismatched Version try to infer the version from python --version to show nice error message

### CODE REVIEW

File: Activities/Database/UiPath.Database/DatabaseConnection.cs [hunk 1]
```diff
@@ -111,13 +113,18 @@ private void SetupCommand(string sql, Dictionary<string, Tuple<object, ArgumentD
...
+            var ceilVal = (int)Math.Ceiling((double)commandTimeout / 1000);
 
             if (ceilVal != 0)
             {
                 _command.CommandTimeout = ceilVal;
-            } 
-            
+            }
+            //Oracle has different implementation for Odbc Stored Procedure calling
+            if (_connection.GetType() == typeof(OdbcConnection)
```
Comment (adrian-zanescu) [inline Activities/Database/UiPath.Database/DatabaseConnection.cs:123]: this is not a python fix. split these in 2 different prs

File: Activities/Python/UiPath.Python/Properties/UiPath.Python.fr.resx [hunk 2]
```diff
@@ -126,4 +126,7 @@
   <data name="PythonRuntimeTypeException" xml:space="preserve">
     <value>Type/méthode/propriété runtime de Python introuvable : {0}</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="UnsupportedVersionException" xml:space="preserve">
```
Comment (adrian-zanescu) [inline Activities/Python/UiPath.Python/Properties/UiPath.Python.fr.resx:129]: remove this from the commit

File: Activities/Python/UiPath.Python/Version.cs [hunk 3]
```diff
@@ -59,11 +62,22 @@ internal static string GetAssemblyName(this Version version)
             return attr.AssemblyName;
         }
 
-        internal static Version Get(this FileVersionInfo fileVersion)
+        internal static Version GetVersionFromStr(this string fileVersion)
```
Comment (adrian-zanescu) [inline Activities/Python/UiPath.Python/Version.cs:65]: a unit test would be nice

**Review by adrian-zanescu — CHANGES_REQUESTED** [review]

File: Activities/Python/UiPath.Python/Properties/UiPath.Python.fr.resx (hunk 2 above)
Comment (petcua1) [reply Activities/Python/UiPath.Python/Properties/UiPath.Python.fr.resx:129]: Was there by mistake. I have removed

**Review by petcua1 — COMMENTED** [review]
All the changes have been resolved

**Review by adrian-zanescu — CHANGES_REQUESTED** [review]

File: Activities/Python/UiPath.Python/Version.cs (hunk 3 above)
Comment (adrian-zanescu) [reply Activities/Python/UiPath.Python/Version.cs:69]: still no tests

**Review by adrian-zanescu — APPROVED** [review]

**Pull request merged**